uber/models/mivs.py

```diff
@@ -100,6 +100,19 @@ class IndieStudio(MagModel):
 
     email_model_name = 'studio'
 
+    @property
+    def primary_contact_first_names(self):
```

Comment:
This is a port of the 'salutation' property from the MITS plugin. The main difference, besides the name, is that we do extra handling of different pluralities of studio members.